src/main/resources/swagger/api-docs.yaml

```diff
@@ -221,6 +221,51 @@ paths:
             - profile
 
   '/api/cluster/{googleProject}/{clusterName}':
+    patch:
+      summary: Updates the configuration of a cluster. Cluster must be running.
+      description:
+      operationId: updateCluster
+      tags:
+        - cluster
+      parameters:
+        - in: path
+          name: googleProject
+          description: googleProject
+          required: true
+          type: string
+        - in: path
+          name: clusterName
+          description: clusterName
+          required: true
+          type: string
+        - in: body
+          description: Request for new cluster
+          name: clusterRequest
+          required: true
+          schema:
+            $ref: '#/definitions/UpdateClusterRequest'
+      responses:
+        '200':
```

Comment:
i should probably make this 202 accepted because update _can_ be async

Reply:
👍